services/QuillLMS/app/workers/save_user_pack_sequence_items_worker.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class SaveUserPackSequenceItemsWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: SidekiqQueue::DEFAULT,
+    lock: :until_executed
+
+  def perform(classroom_id, user_id)
+    return if classroom_id.nil? || user_id.nil?
```

Comment:
Take or leave: if these guard clauses are important, should they be in `UserPackSequenceItemSaver.run`?